src/main/java/com/cinema_booking/booking/movie/model/Movie.java

```diff
@@ -0,0 +1,42 @@
+package com.cinema_booking.booking.movie.model;
+
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import com.cinema_booking.booking.room.model.Room;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
Using @Data in Entity is a bad practice, I would say to stay with the same approach as I mentioned before.